### DISCUSSION TITLE
Show upvoted/downvoted hint on top of card with colour customization

### DISCUSSION THREAD

**Discussion by ghost** [Docile-Alligator/Infinity-For-Reddit#880] [Ideas]
RIF shows whether if the post is already upvoted/downvoted on top AND bottom of card, removing the unnecessary reading the post title, texts, and images then to discover the post is already read at the bottom where the upvote/downvote button is.

Card Layout 2 is not an option to solve this problem as it makes the layout twice as worse, and the upvoted/downvoted hint is still at the bottom of the card.

I use reddit on multiple devices, even on web. The marked posts aren't synced across devices. The internal mark read feature is imperfect for this scenario which I turn off for other reasons besides this. The only way is to rely on the upvoted/downvoted as hint and is consistent comfortably across.

Thank you!

**Comment by Wladefant** [comment]
That is an interesting feature, that could be implemented.
Anyone who is interested in this feature please consider upvoting to show that you are interested
